README.md

```diff
@@ -117,10 +128,25 @@ You can also help out by translating Finamp using our [weblate page](https://hos
 #### Downloads
 </summary>
 
+TODO :)
 <img src=".">
 </details>
 
 
+## Other install methods
```

Comment:
That's nice. You should put a link to this heading somewhere below the install badges at the top :)  
And it's `installation methods` ^^